src/EnergyPlus/SystemAvailabilityManager.hh

```diff
@@ -84,6 +84,9 @@ namespace SystemAvailabilityManager {
 	extern int const HybridVentMode_Enth; // Enthalpy control
 	extern int const HybridVentMode_DewPoint; // Dew point control
 	extern int const HybridVentMode_OA; // Outdoor air control
+	extern int const HybridVentMode_OperT80; // Operative temperature control with 80% acceptability limits
```

Comment:
@mjwitte @Myoldmopar If you agree, I can make changes. The only advantage using extern is that these variables are available in unit tests, if they are never used outside.  

Or we can make a GitHub issue to examine use of extern throughout the whole code. I think other modules may have similar issues.